bouncer/views.py

```diff
@@ -24,6 +26,7 @@ def annotation(self):
                 doc_type="annotation",
                 id=self.request.matchdict["id"])
         except exceptions.NotFoundError:
+            statsd.incr("views.annotation.404.annotation_not_found")
```

Comment:
Will gunicorn tell us the difference between a generic 404 vs an annotation not found?